doc/classes/NavigationAgent2D.xml

```diff
@@ -111,6 +111,21 @@
 		<member name="avoidance_enabled" type="bool" setter="set_avoidance_enabled" getter="get_avoidance_enabled" default="false">
 			If [code]true[/code] the agent is registered for an RVO avoidance callback on the [NavigationServer2D]. When [method NavigationAgent2D.set_velocity] is used and the processing is completed a [code]safe_velocity[/code] Vector2 is received with a signal connection to [signal velocity_computed]. Avoidance processing with many registered agents has a significant performance cost and should only be enabled on agents that currently require it.
 		</member>
+		<member name="debug_enabled" type="bool" setter="set_debug_enabled" getter="get_debug_enabled" default="false">
+			If [code]true[/code] shows debug visuals for this agent.
```

Comment:
As mentioned before - I don't like the fact that agent debug requires global `Visible Navigation` option to be ON to work, but anyway - in that case IMO we should mention that requirement here - something like _if `true` and `Debug/Visible Navigation` is checked, debug visuals will be drawn for this agent_

Reply:
I agree it should not be tied to the Editor->Debug->Visible Navigation that also toggles on the navmesh debug.

That is currently more a GUI band-aid to toggle on the NavigationServer3D `debug_enabled`. Before, this pr did not check for this global debug so if users disabled the global debug the NavigationAgent would still show debug which can also not be the intended behavior.

Can make the NavigationAgent `debug_enabled` toggle on the debug on the NavigationServer when used, this way users can toggle the debug globally after, at least until the debug GUI is improved.